cloud/fabfile.py

```diff
@@ -215,7 +245,7 @@ def get_machines(environment=None):
 
 def deploy(description=None):
     """ [deploy] Make a deployment to an environment. """
-    branch = get_bookmark()
+    branch, summary, version = _get_versioning_metadata()
```

Comment:
by convention you might use underscore as a placeholder for the tuple values you don't care about in this particular case: `branch, _, _ = _get_versioning_metadata()`. Same comment applies below too.